docs/UserGuide.md

```diff
@@ -279,6 +295,23 @@ For example:
 
   ![Reset after](images/ugimages/ResetAfter.png)
 
+#### Recalling previously entered commands
+
+If you have to enter a similar command repeatedly, you can use this feature to type less.
+
+When the [command box](#using-tutors-pet) is selected, press the <kbd>↑</kbd> and <kbd>↓</kbd> keys to cycle through previously entered commands.
+
+For example:
+
+* You are taking the attendance for one of your lessons, CS2103T Tutorial.
```

Comment:
Would it be better to change this to numbered list instead of bullet points?

Reply:
I did consider changing the bullet points to numbered steps, however I decided against it since its the mock PE tomorrow and its unlikely that everyone will be able to standardise it by then.